src/utils.ts

```diff
@@ -119,6 +119,53 @@ export const debounce = <T extends any[]>(
   return ret;
 };
 
+// throttle callback to execute once per animation frame
+export const throttleRAF = <T extends any[]>(fn: (...args: T) => void) => {
+  let handle: number | null = null;
+  let lastArgs: T | null = null;
+  let callback: ((...args: T) => void) | null = null;
+  const ret = (...args: T) => {
+    if (process.env.NODE_ENV === "test") {
+      fn(...args);
+      return;
+    }
```

Comment:
I've disabled the rAF in tests as we'd have to rewrite a lot of them (we're often manually triggering multiple `pointermove` events to test several things within a single test case).

I don't think testing the rAF behavior would yield anything useful, but I may be wrong.